api/server/httputils/form.go

```diff
@@ -49,6 +49,16 @@ type ArchiveOptions struct {
 	Path string
 }
 
+type validationError struct {
+	Err error
+}
+
+func (e validationError) Error() string {
+	return e.Err.Error()
```

Comment:
This is fine, but we still use this in two other places where it's a different error message.